index.js

```diff
@@ -41,26 +40,26 @@ program
 console.log('Open Translators to Things CLI:');
 console.log('');
 
-if(program.menu) {
+if (program.menu) {
     prompt({ currentController: new MainController(), controllerStack: [] });
 }
 
 else if (program.onboarding) {
-    if (program.name === undefined) {
-        console.log("Need to provide name, -n <name>");
+    if (program.hub === undefined) {
+        console.log("Need to provide hub name, -h <hub name>");
         return;
     }
 
     console.log("------ Doing onboarding for %j".header, program.onboarding);
```

Comment:
There's a lot of legacy logging code in here that should be cleaned up.  I'd prefer to do that as it's own change.

---
In reply to: [103610447](https://github.com/openT2T/opent2t-cli/pull/24#discussion_r103610447) [](ancestors = 103610447)